src/Angor/Avalonia/Angor.Wallet/Domain/BroadcastedTransaction.cs

```diff
@@ -0,0 +1,15 @@
+namespace Angor.Wallet.Domain;
+
+public record BroadcastedTransaction(
```

Comment:
Can you explain what is a broadcased trx in this context? 
and why we need it to be a seperated object, is it related to the calculations of the pending payments?